.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -73,35 +86,73 @@ steps:
       -e ARCHITECTURE=${ARCHITECTURE} \
       ${CONTAINER_IMAGE} make -f packer.mk test-building-vhd
     displayName: Run VHD cache test
+
   - task: PublishPipelineArtifact@0
     inputs:
         artifactName: 'vhd-release-notes-${{ parameters.artifactName }}'
         targetPath: 'release-notes.txt'
+
   - script: |
       echo MODE=$(MODE) && \
       OS_DISK_SAS="$(cat packer-output | grep -a "OSDiskUriReadOnlySas:" | cut -d " " -f 2)" && \
       VHD_NAME="$(echo $OS_DISK_SAS | cut -d "/" -f 8 | cut -d "?" -f 1)" && \
       docker run --rm \
       -v ${PWD}:/go/src/github.com/Azure/AgentBaker \
       -w /go/src/github.com/Azure/AgentBaker \
-      -e CLIENT_ID=${CLIENT_ID} \
-      -e CLIENT_SECRET="$(CLIENT_SECRET)" \
-      -e TENANT_ID=${TENANT_ID} \
-      -e SUBSCRIPTION_ID="${SUBSCRIPTION_ID}" \
+      -e CLIENT_ID=${BUILD_CLIENT_ID} \
+      -e CLIENT_SECRET="$(BUILD_CLIENT_SECRET)" \
```

Comment:
why do you change these? Currently Windows uses `CLIENT_ID`, `CLIENT_SECRET` and `TENANT_ID` to copy the built VHD to the specified storage account and it works.

Reply:
It didn't work before. Will check on this.

Reply:
I misunderstood. I think `CLIENT_ID`, `CLIENT_SECRET`, and `TENANT_ID` are set in containerd&docker pipeline, but not in the PR check-in pipeline. When I was incorrectly using DRY_RUN=False and toying around with az-copy and convert-sig-to-classic-storage-account, I set them to `BUILD_...`. Corrected in the commit https://github.com/Azure/AgentBaker/pull/2036/commits/7dfa348264f6453af6a2be809c1b517b66b94de8.